content/en/privacy.md

```diff
@@ -32,7 +32,7 @@ You may need to download client software from a repository such as those run by
 
 ## Visitor
 
-When you are a Visitor browsing the Let’s Encrypt web site, you have the option to make a donation. Donations are processed by our trusted payment partners including DonorBox, Stripe, Shopify, and PayPal, depending on the payment method selected. We collect your name and email address when you donate. We will not use your email address to contact you without your consent unless we feel it's necessary to resolve an issue related to a particular donation. Your interactions with [DonorBox](https://donorbox.org/privacy), [Stripe](https://stripe.com/privacy/), and [PayPal](https://www.paypal.com/us/webapps/mpp/ua/privacy-full) are governed by their respective privacy policies. We do not collect or retain any credit card or bank information related to donations. If we collect a physical address, we will only retain your physical address information for as long as is reasonably necessary to make the shipment that you requested.
+When you are a Visitor browsing the Let’s Encrypt web site, you have the option to make a donation. Donations are processed by our trusted payment partners including DonorBox, Stripe, Shopify, and PayPal, depending on the payment method selected. We collect your name and email address when you donate. We will not use your email address to contact you without your consent unless we feel it's necessary to resolve an issue related to a particular donation. Your interactions with [DonorBox](https://donorbox.org/privacy), [Stripe](https://stripe.com/privacy/), [PayPal](https://www.paypal.com/us/webapps/mpp/ua/privacy-full), [Shopify](https://www.shopify.com/legal/privacy), and [SHIPHERO](https://shiphero.com/privacy-data-policy/) are governed by their respective privacy policies. We do not collect or retain any credit card or bank information related to donations. If we collect a physical address, we will only retain your physical address information for as long as is reasonably necessary to make the shipment that you requested via Shopify and our fulfillment partner, SHIPHERO.
```

Comment:
Could you please remove 
> , [Shopify](https://www.shopify.com/legal/privacy), and [SHIPHERO](https://shiphero.com/privacy-data-policy/)

from the current location and add it at the end, like this? 

> If we collect a physical address, we will only retain your physical address information for as long as is reasonably necessary to make the shipment. Our fulfillment partners are Shopify and SHIPHERO. Your interactions with [Shopify](https://www.shopify.com/legal/privacy) and [SHIPHERO](https://shiphero.com/privacy-data-policy/) are governed by their respective privacy policies.

Reply:
@SarahGran updated, let me know if it looks good to go!